tasks/Module 4/task_1.py

```diff
@@ -0,0 +1,27 @@
+"""Mad libs."""
+import os
+import re
+
+TEMPLATE = ['ADJECTIVE', 'NOUN', 'ADVERB', 'VERB']
+
+
+def mad_lib(source_file_path):
+    """Repace placeholders with user inputs."""
+    tmp_file = source_file_path + '.tmp'
+    with open(source_file_path, 'r') as read_file:
```

Comment:
You can do it even in one line if it fits your style:
```python
with open(source_file_path, 'r') as read_file, open(tmp_file, 'w') as write_file:
```

Reply:
I afraid it would be more then 79 characters and linters will force me to break it into 2 lines.

Reply:
Ah, historical standards ;-) Nowadays people tend to not care too much about it 79 characters. Usually real world projects use 100 or even 120 characters limit.

If you don't like this default limit you can change it to something else by creating `.pylintrc` file in root of your repository and adding following content:
```ini
[FORMAT]
max-line-length=100
```

Reply:
wow. it would be really nice.
Thank you.